main/src/ca/uwaterloo/flix/language/phase/MonoTypes.scala

```diff
@@ -437,7 +437,7 @@ object MonoTypes {
     */
   private def specializeEnum(sym: Symbol.EnumSym, args0: List[Type], loc: SourceLocation)(implicit ctx: SharedContext, root: LoweredAst.Root, flix: Flix): Symbol.EnumSym = {
     // type arguments
-    val args = args0.map(eraseAliases).map(TypeNormalization.normalizeType)
+    val args = args0.map(eraseAliases).map(eraseEffects).map(TypeNormalization.normalizeType)
```

Comment:
Only here? Nowhere else?

What happens when this phase is deleted?

Reply:
Indeed, the rules for erasure and normalization are not totally clear. It's a problem that pervades these two phases, hence Jonathan's PR with `NormalType` etc.